apps/cookbook/src/app/examples/accordion-example/accordion-example.component.scss

```diff
@@ -1 +1,6 @@
 @use '../examples.shared';
+@use '@kirbydesign/core/src/scss/utils';
+
+.example-frame {
```

Comment:
Consider calling this `.xl-padding` or something, and adding it to the shared `example-frame` class next to `no-padding`, so we can reuse across.